packages/nodes-base/nodes/Slack/MessageDescription.ts

```diff
@@ -487,7 +487,7 @@ export const messageFields: INodeProperties[] = [
 		displayName: 'Text',
 		name: 'text',
 		type: 'string',
-		required: true,
+		required: false,
```

Comment:
Why message was changed to not required? Please follow the same rules we have in the message:post operation.

Reply:
This is the snipped from message:post in which Text is not required
```
	{
		displayName: 'Text',
		name: 'text',
		type: 'string',
		typeOptions: {
			alwaysOpenEditWindow: true,
		},
		default: '',
		displayOptions: {
			show: {
				operation: [
					'post',
					'postEphemeral',
				],
				resource: [
					'message',
				],
			},
		},
		description: 'The text to send.',
	},
```

Slack permits either text or blocks or both

Reply:
https://api.slack.com/methods/chat.update#arg_text

Directly from the Slack docs:

> New text for the message, using the [default formatting rules](https://api.slack.com/reference/surfaces/formatting). It's not required when presenting blocks or attachments.